staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -400,6 +412,10 @@ func (le *LeaderElector) Check(maxTolerableExpiredLease time.Duration) error {
 	return nil
 }
 
+func (le *LeaderElector) observedRecordOnLease(now time.Time) bool {
```

Comment:
This name is super misleading - maybe `isLeaseValid` or sth like that?

Reply:
Done